restapi/src/main/java/io/stargate/web/docsapi/exception/ErrorCode.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.web.docsapi.exception;
+
+import io.stargate.web.models.Error;
+import java.util.Optional;
+import javax.ws.rs.core.Response;
+
+/** Simple enumeration for the error code that can provide back the correct response to the user. */
+public enum ErrorCode {
+
+  /** Generic data store errors. */
+  DATASTORE_KEYSPACE_DOES_NOT_EXIST(Response.Status.NOT_FOUND, "An unknown namespace provided."),
+  DATASTORE_TABLE_DOES_NOT_EXIST(Response.Status.NOT_FOUND, "An unknown table provided."),
+  DATASTORE_TABLE_NAME_INVALID(
+      Response.Status.BAD_REQUEST,
+      "The table name contains invalid characters. Valid characters are alphanumeric and underscores."),
+
+  /** Document API. */
+  DOCS_API_GENERAL_PAGE_SIZE_EXCEEDED(
+      Response.Status.BAD_REQUEST, "The parameter `page-size` is limited to 20.");
```

Comment:
Could we have a constant for `20` so that the code using the error can be sure to use the same value in its checks?

Reply:
I don't think so, this has to be generic for all error codes?

Reply:
yep that's doable.. thanks for clarification :+1:.. Adding..